pkg/git/draft.go

```diff
@@ -63,9 +64,15 @@ func (d *gitPackageDraft) UpdateLifecycle(ctx context.Context, new v1alpha1.Pack
 }
 
 // Finish round of updates.
-func (d *gitPackageDraft) Close(ctx context.Context) (repository.PackageRevision, error) {
+func (d *gitPackageDraft) Close(ctx context.Context, version string) (repository.PackageRevision, error) {
```

Comment:
This should not allow empty strings, or it should default to v1. Probably better to not allow empty strings.

Reply:
Hey Greg, I've added a check to fail if the version is empty now.